### ISSUE TITLE
Automatic keyboard navigation setting should have a markdown description

### ISSUE THREAD

**Issue by roblourens** [Microsoft/vscode#69488]
![image](https://user-images.githubusercontent.com/323878/53454875-82693780-39dd-11e9-8038-4efaeebdec1f.png)

You can use `markdownDescription` so backticks will be rendered as code
